lci_strategic_plugin/include/lci_strategic_plugin/lci_strategic_plugin.h

```diff
@@ -513,6 +513,22 @@ class LCIStrategicPlugin
                                                                      bool shortest_path_only = false,
                                                                      bool bounds_inclusive = true) const;
   
+  /**
+   * \brief Get the final entry time from either vehicle's own internal ET calculation (TSMO UC2) or from carma-street (TSMO UC3)
+   *        This function also applies necessary green_buffer to adjust the ET. 
+   * \param current_state Current state of the vehicle
+   * \param earliest_entry_time Earliest entry time calculated by the vehicle
+   * \param traffic_light traffic signal the vehicle is using
+   * \param is_entry_time_within_green_or_tdb this function sets it to true if ET is in green or TDB (related to UC3, always set to true in UC2)
```

Comment:
Minor: Should, be TBD (**T**o **B**e **D**etermined) I believe

Reply:
This is my bad, I keep interchanging the two. I changed all instances